charts/azuremonitor-containers/values.yaml

```diff
@@ -65,6 +65,7 @@ omsagent:
   proxy: <your_proxy_config>
   # This metricsEndpoint used to define the endpoint custom metrics emit to. If not defined, default public Azure monitoring endpoint '{aks_region}.monitoring.azure.com' will be used.
   metricsEndpoint: <your_metrics_endpoint>
+  resourceEndpoint: <your_resource_endpoint>
```

Comment:
For Arc K8s, the audience specified here - https://github.com/microsoft/Docker-Provider/blob/ci_prod/charts/azuremonitor-containers/templates/ama-logs-arc-k8s-crd.yaml#L22 which will indicate the Arc K8s extension platform to get the token with the specified audience.

Reply:
Thanks for the information! Does this mean the corresponding Ruby code will be changed to get IMDS token from K8s CRD later?

Reply:
Hi, @ms-hujia ,  The audience which we have is fixed. whatever the audience defined in the CRD matches with ruby code.  In your case, if the audience is custom, in that case, ruby code should use the same audience defined in the CRD.   do you have the steps which you have followed to validate this change?